tests/integration/ledger/secpinterface.go

```diff
@@ -0,0 +1,11 @@
+package ledger
+
+type SECP256K1 interface {
```

Comment:
Is there a reason to re-define this rather than to use the Cosmos SDK's declaration?

Reply:
@4rgon4ut let's use the interface from the SDK

Reply:
@fedekunze @austinchandra This one was created to create mock with mockery tool, I will just remove it